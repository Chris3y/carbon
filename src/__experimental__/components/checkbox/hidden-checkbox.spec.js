import React from 'react';
import TestRenderer from 'react-test-renderer';
import 'jest-styled-components';
import HiddenCheckbox from './hidden-checkbox.component';

function render(props) {
  return TestRenderer.create(<HiddenCheckbox { ...props } />);
}

describe('HiddenCheckbox', () => {
  it('renders as expected', () => {
    expect(render()).toMatchSnapshot();
  });
});
