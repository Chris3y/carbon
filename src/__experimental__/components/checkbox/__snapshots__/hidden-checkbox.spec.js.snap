// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HiddenCheckbox renders as expected 1`] = `
.c0 {
  cursor: pointer;
  opacity: 0;
  margin: 0;
  z-index: 999;
}

<input
  className="c0"
  role="checkbox"
  type="checkbox"
/>
`;
